client.go

```diff
@@ -235,16 +239,28 @@ func (c *Client) Queues() ([]*Queue, error) {
 	for _, jobName := range jobNames {
 		count, err := redis.Int64(conn.Receive())
 		if err != nil {
-			logError("client.queues.receive", err)
+			logError("client.queues.receive.count", err)
 			return nil, err
 		}
 
-		queue := &Queue{
-			JobName: jobName,
-			Count:   count,
+		maxConcurrency, err := redis.Int64(conn.Receive())
```

Comment:
What happens here if key does not exist?

Reply:
Changed to not error out when `redis.ErrNil` is returned.